### PR TITLE
Edit the Node template with mobifi in place of node-template

### PR DESCRIPTION
**Pull request by aelesbao**
Renamed the runtime, node and template pallet.

_Open questions:_

- Should we configure the Github workflow now?
- Which standard should we use for the modules, `Mobifi` or `MobiFi`?

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -95,8 +95,8 @@ pub mod opaque {
 // To learn more about runtime versioning and what each of the following value means:
 //   https://substrate.dev/docs/en/knowledgebase/runtime/upgrades#runtime-versioning
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("node-template"),
-	impl_name: create_runtime_str!("node-template"),
+	spec_name: create_runtime_str!("mobifi"),
+	impl_name: create_runtime_str!("mobifi"),
```
Comment (aelesbao) [inline runtime/src/lib.rs:99]: Should we use `mobifi` or `mobifi-node` here?

File: node/Cargo.toml [hunk 2]
```diff
@@ -1,16 +1,16 @@
 [package]
-authors = ['Substrate DevHub <https://github.com/substrate-developer-hub>']
+authors = ['MobiFi <info@mobifi.io>']
```
Comment (nblogist) [inline node/Cargo.toml:2]: please change it to DotMatrix @aelesbao

File: pallets/mobifi/Cargo.toml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 [package]
-authors = ['Substrate DevHub <https://github.com/substrate-developer-hub>']
-description = 'FRAME pallet template for defining custom runtime logic.'
+authors = ['MobiFi <info@mobifi.io>']
```
Comment (nblogist) [inline pallets/mobifi/Cargo.toml:2]: same here @aelesbao

File: runtime/Cargo.toml [hunk 4]
```diff
@@ -1,11 +1,11 @@
 [package]
-authors = ['Substrate DevHub <https://github.com/substrate-developer-hub>']
+authors = ['MobiFi <info@mobifi.io>']
```
Comment (nblogist) [inline runtime/Cargo.toml:2]: please change in any other files as well @aelesbao

File: runtime/src/lib.rs (hunk 1 above)
Comment (nblogist) [reply runtime/src/lib.rs:99]: let's stick with the language schematics.

**Comment by nblogist** [comment]
Thanks @aelesbao  I think we can keep it `MobiFi` where we language schematics allow to capitalize, inother places `mobifi` should be fine

File: runtime/src/lib.rs [hunk 5]
```diff
@@ -482,7 +480,7 @@ impl_runtime_apis! {
 			add_benchmark!(params, batches, frame_system, SystemBench::<Runtime>);
 			add_benchmark!(params, batches, pallet_balances, Balances);
 			add_benchmark!(params, batches, pallet_timestamp, Timestamp);
-			add_benchmark!(params, batches, pallet_template, TemplateModule);
+			add_benchmark!(params, batches, pallet_mobifi, MobifiModule);
```
Comment (nblogist) [inline runtime/src/lib.rs:483]: I think `MobiFiModule` would suit better here, what do you think @aelesbao ?

File: runtime/src/lib.rs (hunk 1 above)
Comment (nblogist) [reply runtime/src/lib.rs:99]: see how others are doing?

Comment (aelesbao) [reply runtime/src/lib.rs:99]: Based on the research I did, there's not much consensus about it. Also, [looking at the documentation](https://substrate.dev/rustdocs/v3.0.0/sp_version/struct.RuntimeVersion.html#structfield.impl_name), it seems like the `impl_name` _is of little consequence for the node and serves only to differentiate the code of different implementation teams_. I checked [Acala's repository](https://github.com/AcalaNetwork/Acala/blob/master/runtime/acala/src/lib.rs#L116) and they are using the same name on both, so it should be fine to keep it like that.

**Comment by nblogist** [comment]
Looks good, we can later change the pallet to add the light client in there.

File: runtime/src/lib.rs (hunk 1 above)
Comment (nblogist) [reply runtime/src/lib.rs:99]: Alright @aelesbao

**Pull request merged**